src/core/util/sys_vars.cpp

```diff
@@ -297,9 +297,9 @@ const char *to_str(MODE option, const OPT (&options)[N])
     }
 
 #define AUTO_ON_OFF_IMPL                                                                           \
-    {AUTO, "Auto", {"auto", NULL, NULL}}, {ON, "Enabled", {"on", "enabled", NULL}},                \
+    {AUTO, "Auto", {"auto", NULL, NULL}}, {ON, "true", {"on", "enabled", NULL}},                \
     {                                                                                              \
-        OFF, "Disabled",                                                                           \
+        OFF, "false",                                                                           \
         {                                                                                          \
             "off", "disabled", NULL                                                                \
         }                                                                                          \
```

Comment:
Looks inconsistent to me. Since the main point of the change is to unify reading and printing the parameters, lets either address this block completely or keep it as is for now and improve it later.

Is it used only for legacy config subsystem? We don't have to rework something that will be removed soon. If the code remains, we can discuss how to make it consistent with JSON schema.